### PR TITLE
Updated to Paket, Integrated into origin repo

### PR DESCRIPTION
**Pull request by mizzle-mo**
The paket deps may not be right.  We might have to pull some of the "fable-" stuff out of package,json and move it into paket.  Let me know if you have any questions.

### CODE REVIEW

File: src/bindings/bindings.fsproj [hunk 1]
```diff
@@ -49,7 +49,6 @@
   <ItemGroup>
     <Compile Include="monaco.fs" />
     <Compile Include="virtualdom.fs" />
-    <Compile Include="babel.fs" />
```
Comment (tpetricek) [inline src/bindings/bindings.fsproj:52]: Is this intentionally removed?

**Comment by tpetricek** [comment]
Thanks very much for getting started with this!

I had a quick look at the diff and it seems that some of the changes I did recently in the `master` branch are not reflected in your version - could you make sure that you are doing this against the latest version? (I imagine that just copying all the `fs` files that you did not intend to modify from master into your copy should do the trick!)

I marked one above - but it seems that changes in `interactive.fs` and many of the other code changes are an error.

**Comment by mizzle-mo** [comment]
Ok Fable is stable now, so I think we should redo this based on the current template.  I'm going to work on this today.  Sorry for the delays, real life keeps trying to knock me around.

**Comment by mizzle-mo** [comment]
Closing this and will resubmit off a fresh fork.

**Pull request closed without merging**